<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_mess</class>
 <widget class="QDialog" name="User_mess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(18, 53, 85);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 17pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>尊敬的会员 您好 当前用餐享受8.8折</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>38</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 15pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>57</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 15pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>卡号 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_money">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 15pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>余额</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_credits">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 15pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>积分</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_history_time">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>57</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 15pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_history_order">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>57</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 15pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>菜品</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_history_money">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>57</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 15pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>金额</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_changepsw">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(225, 238, 210);
font: 14pt &quot;Sans Serif&quot;;
color: rgb(230, 155, 3);</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(225, 238, 210);
font: 14pt &quot;Sans Serif&quot;;
color: rgb(230, 155, 3);</string>
   </property>
   <property name="text">
    <string>退出账户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_order">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>440</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(225, 238, 210);
font: 14pt &quot;Sans Serif&quot;;
color: rgb(230, 155, 3);</string>
   </property>
   <property name="text">
    <string>开始点餐</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_history_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>91</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 238, 210);
color: rgb(18, 53, 85);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_history_order">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 238, 210);
color: rgb(18, 53, 85);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_history_money">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>61</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 238, 210);
color: rgb(18, 53, 85);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>201</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_name_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(230, 155, 3);
font: 15pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_CID">
      <property name="styleSheet">
       <string notr="true">color: rgb(230, 155, 3);
font: 15pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_balance">
      <property name="styleSheet">
       <string notr="true">color: rgb(230, 155, 3);
font: 15pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_integral">
      <property name="styleSheet">
       <string notr="true">color: rgb(230, 155, 3);
font: 15pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Button_bind">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(225, 238, 210);
font: 14pt &quot;Sans Serif&quot;;
color: rgb(230, 155, 3);</string>
   </property>
   <property name="text">
    <string>绑定卡号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Recharge">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(225, 238, 210);
font: 14pt &quot;Sans Serif&quot;;
color: rgb(230, 155, 3);</string>
   </property>
   <property name="text">
    <string>充值</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
